africanus/model/coherency/tests/test_convert.py

```diff
@@ -51,7 +51,7 @@ def visibility_factory(vis_shape, input_shape, in_type, backend="numpy", **kwarg
     else:
         raise ValueError("Invalid backend %s" % backend)
 
-    if in_type == "real":
+    if in_type == "complex":
```

Comment:
Maybe in_type is redundant, now that the tests aren't distinguishing between complex and real? Could it be removed from the cases and `visibility_factory`?